server/main.lua

```diff
@@ -115,14 +108,15 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job, cityhallCoords)
     if not Player then return end
     local ped = GetPlayerPed(src)
     local pedCoords = GetEntityCoords(ped)
+
     local data = {
         ["src"] = src,
         ["job"] = job
     }
     if #(pedCoords - cityhallCoords) >= 20.0 or not availableJobs[job] then
         return false -- DropPlayer(source, "Attempted exploit abuse")
     end
-    if QBCore.Shared.QBJobsStatus then exports["qb-jobs"]:submitApplication(data)
+    if QBCore.Shared.QBJobsStatus then exports["qb-jobs"]:submitApplication(data,"Jobs")
```

Comment:
Remember to put a space before passing the second param

Reply:
I blame road, I'll fix it rn

Reply:
It's a space not necessary